### PR TITLE
Restore the rev attribute to <a> and <link>

### PR DESCRIPTION
**Pull request by travisleithead**
Removes it from the obsolete section too and fixes a few links.
See Issue #256.

### CODE REVIEW

File: sections/semantics-document-metadata.include [hunk 1]
```diff
@@ -410,6 +409,30 @@
...
+  <div class="example">Given two documents, each containing a chapter of a book, the links between
+  them could be described with the <{link/rel}> and <{links/rev}> attributes as follows (where both
+  links are semantically equivalent:
+
+  Document with URL "chapter1.html" 
+  <pre highlight="html">&lt;link href="chapter2.html" rel="Next"&gt;</pre>
+  
+  Document with URL "chapter2.html" 
+  <pre highlight="html">&lt;link href="chapter1.html" rev="Prev"&gt;</pre>
+
```
Comment (chaals) [inline sections/semantics-document-metadata.include:454]: Actually, I think the example is wrong.

The relationships defined are **directional**. So if the "Next" document, for doc1, is doc2, then doc2 has a "previous" document which is doc1.

In the example here Chapter1.html would have
`<link rel="Next" rev="Prev" href="Chapter2.html">`
(and Chapter2.html would have
`<link rel="Prev" rev="Next" href="Chapter1.html">`
`<link rel="Next" rev="Prev" href="Chapter3.html">`
by way of symmetry)

A table of contents might have

```
  <li><a href="ch1" rel="Next" rev="toc">chapter 1</a></li>
  <li><a href="ch2" rev="toc">chapter 1</a></li>
  <li><a href="ch3" rev="toc">chapter 1</a></li>
```

Comment (travisleithead) [inline sections/semantics-document-metadata.include:454]: Thanks @chaals. That helps--I'll fix the example tomorrow..

Comment (chaals) [inline sections/semantics-document-metadata.include:454]: OK. Might be useful in the explanation here and at ll 325-328 above to point out that the relationship is _directional_ - which is why `rev` can be useful...

File: sections/semantics-document-metadata.include [hunk 2]
```diff
@@ -410,6 +407,30 @@
     information regarding the license under which the current page is provided.
   </div>
 
+  <{link}> and <{a}> elements may also have a
```
Comment (arronei) [inline sections/semantics-document-metadata.include:410]: What about `<area>` elements? `<area>` should also have `rev` attributes.

File: sections/semantics-document-metadata.include [hunk 3]
```diff
@@ -367,33 +364,33 @@
...
     element.
   </p>
 
   The types of link indicated (the relationships) are given by the value of the
-  <dfn element-attr for="link"><code>rel</code></dfn> attribute, which, if present, must have a value that is a <a>set of
-  space-separated tokens</a>. The <a>allowed keywords and their meanings</a> are defined in a later
-  section. <span class="impl">If the <code>rel</code> attribute is absent, has no keywords, or if
-  none of the keywords used are allowed according to the definitions in this specification, then the
-  element does not create any links.</span>
+  <dfn element-attr for="link,links"><code>rel</code></dfn> attribute, which, if present, must have a
```
Comment (arronei) [inline sections/semantics-document-metadata.include]: should the `for=` actually be `a,area,link`?

File: sections/semantics-document-metadata.include [hunk 4]
```diff
@@ -410,6 +407,30 @@
     information regarding the license under which the current page is provided.
   </div>
 
+  <{link}> and <{a}> elements may also have a
+  <dfn element-attr for="links,link,a"><code>rev</code></dfn> attribute, which is used to describe
```
Comment (arronei) [inline sections/semantics-document-metadata.include:411]: Should the `for=` actually be `a,area,link,links`?

File: sections/semantics-links.include [hunk 5]
```diff
@@ -77,28 +77,29 @@
   what punctuation is supported in file names, and user agents are likely to adjust file names
   accordingly.
 
-  The <dfn element-attr for="links"><code>rel</code></dfn> attribute on <{a}> and
+  The <dfn element-attr for="a,area"><code>rel</code></dfn> attribute on <{a}> and
```
Comment (arronei) [inline sections/semantics-links.include:80]: Is this `rel` specifically for `a` and `area`?

File: sections/semantics-document-metadata.include (hunk 2 above)
Comment (travisleithead) [inline sections/semantics-document-metadata.include:410]: No, in HTML4, neither `rel` nor `rev` are allowed in `<area>`.

File: sections/semantics-document-metadata.include (hunk 4 above)
Comment (travisleithead) [inline sections/semantics-document-metadata.include:411]: Nope. See above comment.

File: sections/semantics-links.include (hunk 5 above)
Comment (travisleithead) [inline sections/semantics-links.include:80]: I cross-checked with our sister spec--apparently it is.

**Comment by travisleithead** [comment]
OK, relatively major revision. Please take another look.

**Comment by arronei** [comment]
+1

@chaals you had comments before can you confirm that your issues are addressed?

**Comment by travisleithead** [comment]
Additional changes from the discussion in Issue #256...

**Comment by danbri** [comment]
LGTM

**Pull request merged**